src/modules/curriculum/CurriculumInformation.js

```diff
@@ -70,30 +70,31 @@ class CurriculumInformation extends Component {
         <InfoColumn>
           <Tout title={'Upcoming Retreats'}>
             <p>
-              <span className="bold">February 10 – 23, 2019</span>
-              <br />Deposit payments are due October 31, 2018. Complete your
-              registration by December 15, 2018 to be eligible for our Early
-              Bird Pricing!
+              <span className="bold">October 27 – November 6, 2019</span>
+              <br />
+              Complete your registration by May 31st, 2019 to be eligible for
+              our Early Bird Pricing!
```

Comment:
Thoughts on bolding `Early Bird Pricing`?
```suggestion
              <em>our Early Bird Pricing!</em>
```